src/utils/_forms.scss

```diff
@@ -29,3 +29,84 @@
     display: none;
   }
 }
+
+// Any form field: input, textarea, ...
+// e.g: @include formField(input);
+
+@mixin form-field($field) {
+  overflow: hidden;
+
+  &-#{$field} {
+    border-color: $c-gray-light;
+    border-style: solid;
+    border-width: 1px;
+    font-size: $fz-base;
+    padding-left: $p-l;
+    padding-right: $p-l;
+    width: 100%;
+
+    &:not(:disabled)::placeholder {
+      color: $c-gray-dark;
+    }
+
+    &:disabled {
+      background: $c-gray-lightest;
+
+      &::placeholder {
+        color: $c-gray-light;
+      }
+    }
+
+    &:focus {
+      box-shadow: inset 0 0 3px 0 $c-primary;
+    }
+  }
+
+  //sui-AtomInput-success & sui-AtomInput-error
+  $validation: success $c-success, error $c-error;
+
+  @each $type, $color in $validation {
+    &-#{$type} {
+      label {
+        color: $color;
+      }
+
+      #{$field} {
+        border-color: $color;
+      }
+
+      &-validation-text {
+        color: $color;
+        display: block;
+        font-size: $fz-xs;
+        margin: $m-s 0 0;
+      }
+    }
+  }
+}
+
+// Help text for form elements
+@mixin help-text {
+  color: $c-gray-light;
+  display: block;
+  font-size: $fz-xs;
+  margin: $m-s 0 0;
+}
+
+// Size for form elements
+@mixin field-size($field) {
```

Comment:
if i'm not wrong, this should be used like so:
```scss
.sui-Form {
  @include form-field(input);
  @include field-size(input);
}
```
is this correct?

i was wondering why the mixin `field-size` is not called directly from `form-field`.
so we can just:
```scss
.sui-Form {
  @include form-field(input);
}
```

Reply:
I think it's better split to use by separate

Reply:
not really 😓 
I've moved it and now it's called from `form-field`